web/src/styles/lifecycle.css

```
@@ -53,6 +59,1578 @@ html[data-theme="noir-gold"] {
+  padding: 16px 0;
+  font-style: italic;
+  margin: 0;
+}
+
+.inbox-mail-prose {
+  font-size: 14.5px;
+  line-height: 1.55;
+  color: var(--text);
+  margin: 0 0 8px;
+  white-space: pre-wrap;
+  word-wrap: break-word;
```

Stylelint (17.11.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)